generals/rewards/frequent_asset_reward_fn.py

```diff
@@ -0,0 +1,28 @@
+from generals.core.action import Action
+from generals.core.observation import Observation
+from generals.rewards.common import compute_num_cities_owned, compute_num_generals_owned, is_action_valid
+from generals.rewards.reward_fn import RewardFn
+
+
+class FrequentAssetRewardFn(RewardFn):
```

Comment:
this is very nice, i was thinking about something similar myself. It is a good start on how people should think about reward shaping for this game :)

Reply:
thanks!